### PR TITLE
Document how to add stacktraces to monolog

### PR DESCRIPTION
**Pull request by tarjei**
IMHO this should be in the main section because it is a major DX issue.

### CODE REVIEW

File: logging.rst [hunk 1]
```diff
@@ -357,6 +358,106 @@ specific channel (``app`` by default), you can either :ref:`autowire monolog cha
 or use the ``monolog.logger`` tag  with the ``channel`` property as explained in the
 :ref:`Dependency Injection reference <dic_tags-monolog>`.
 
+Adding stacktraces from exceptions
```
Comment (OskarStark) [inline logging.rst:361]:
```suggestion
Adding Stacktraces from Exceptions
```

File: logging.rst [hunk 2]
```diff
@@ -159,6 +159,7 @@ to write logs using the :phpfunction:`syslog` function:
 This defines a *stack* of handlers and each handler is called in the order that it's
 defined.
 
+
```
Comment (OskarStark) [inline logging.rst:162]:
```suggestion
```

File: logging.rst [hunk 3]
```diff
@@ -357,6 +358,106 @@ specific channel (``app`` by default), you can either :ref:`autowire monolog cha
 or use the ``monolog.logger`` tag  with the ``channel`` property as explained in the
 :ref:`Dependency Injection reference <dic_tags-monolog>`.
 
+Adding stacktraces from exceptions
+----------------------------------
+
+To include stacktraces to your logs, set the `include_stacktraces` option on the "stream" handler to true and include the exception key in your logging statement:
```
Comment (OskarStark) [inline logging.rst:364]:
```suggestion
To include stacktraces to your logs, set the ``include_stacktraces`` option on the "stream" handler to true and include the exception key in your logging statement:
```

File: logging.rst [hunk 4]
```diff
@@ -357,6 +358,106 @@ specific channel (``app`` by default), you can either :ref:`autowire monolog cha
...
+
+
+```php
+$logger->error($exception->getMessage(), ['exception' => $exception]);
+```
```
Comment (OskarStark) [inline logging.rst:369]:
```suggestion
    $logger->error($exception->getMessage(), ['exception' => $exception]);
```

File: logging.rst [hunk 5]
```diff
@@ -357,6 +358,106 @@ specific channel (``app`` by default), you can either :ref:`autowire monolog cha
...
+
```
Comment (OskarStark) [inline logging.rst:370]:
```suggestion
```

File: logging.rst [hunk 6]
```diff
@@ -357,6 +358,106 @@ specific channel (``app`` by default), you can either :ref:`autowire monolog cha
...
+                // still passed *all* logs, and still only logs error or higher
+                'syslog_handler' => [
+                    'type'  => 'syslog',
+                    'level' => 'error',
+                ],
+            ],
+        ]);
+
+
```
Comment (OskarStark) [inline logging.rst:460]:
```suggestion
```

**Comment by tarjei** [comment]
Thanks for reviewing @OskarStark !

**Comment by nicolas-grekas** [comment]
This PR has been closed because the master has been removed.
Please submit it again against the appropriate branch.

**Pull request closed without merging**